PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,30 @@
-## High Level Description
+## High-level description
 
-What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
+What features does this change enable? What bugs does this change fix?
 
-## Related Issues
 
-  - [DCOS_OSS-<number>](https://jira.dcos.io/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.
+## Corresponding DC/OS tickets
+
+These DC/OS JIRA ticket(s) must be updated in the moment this PR lands:
+
+  - https://jira.mesosphere.com/browse/DCOS_OSS-<number>
+
+
+## Related tickets
```

Comment:
We are suggesting users to do **something** with *3* JIRA tickets, in 

**Corresponding DC/OS tickets** section.

These DC/OS JIRA ticket(s) must be updated in the moment this PR lands:

1) https://jira.mesosphere.com/browse/DCOS_OSS-

These tickets are related to this change:

2)  [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.
3) [DCOS-<number>](https://jira.mesosphere.com/browse/DCOS-<number>) Foo the Bar so it stops Bazzing.		    

This is an action requested from user and never enforced automatically, I suggest we simplify, keep only 1) and kill 2) and 3).

Reply:
I agree that it should be clear that what's there are examples. And one example is probably enough in each of both relevant section.

In the second section I have removed one of two examples.